### ISSUE TITLE
Windows 11 BUG 希望您能尽快修复

### ISSUE THREAD

**Issue by Nesxc** [Tatsu-syo/noMeiryoUI#40]
症状描述：在windows11系统下重启，会有一段时间字体为默认字体，大概0.1-0.5分钟时间才会重新应用字体。
软件版本：3.0.0
操作环境：Windows 11 Pro 21H2

这个BUG会让人觉得有点难受，希望您能尽快修复他以达到windows10那样的丝滑流畅，并且感谢您提供了那么好的Software。

: ) @Tatsu-syo

**Comment by iccfish** [comment]
> Hmm, Windows 11's behavior? I think it's not a big issue. But, it's weird.

Yep...it's definitely Windows 11's behavior. I also noticed that after reboot it has high possibility reset to default.

**Comment by Tatsu-syo** [comment]
Did you use font settings file and -set option?
If your system's font settings are reset frequently, please try font settings file and -set option, please try it.

**Comment by iccfish** [comment]
> Did you use font settings file and -set option? If your system's font settings are reset frequently, please try font settings file and -set option, please try it.

yeah... I'm using this command line and -set option to auto apply font settings after system upgrade(I'm in Insider preview fast ring). But it seems that these settings always reset to default after reboot on my laptop.

**Comment by Tatsu-syo** [comment]
Hmm, I don't have Insider preview fast ring environment.
Hmm, should I build insider preview environment?

**Comment by Nesxc** [comment]
> 

不好意思。我没有收到邮件通知，所以才现在回复。

在我看来这是个伟大的软件。他在Windows10上运行完美，但是在Windows11上的问题让人难受，所以，请求您修复这个bug😃

**Comment by iccfish** [comment]
> Hmm, I don't have Insider preview fast ring environment. Hmm, should I build insider preview environment?

I don't think so. The problem was I can not figure out what cause font settings reset to default. It was odd.

Previously I know windows 11 insider preview has some features in test via test flags, and MS can remotly change these features on and off via schedule task (\Microsoft\Windows\Flighting\FeatureConfig\ReconcileFeatures). Currently  I've done made this task disabled. I will check if this can solve this strange behavior.

**Comment by Tatsu-syo** [comment]
@iccfish Thank you.
But why does MS want to reset the settings anyway?

**Comment by iccfish** [comment]
> @iccfish Thank you. But why does MS want to reset the settings anyway?

I guess this was not in design, just cause of when apply thiese personalize settings, font settings was reset to default accidentally due to lack of these settings in target profile.

**Comment by Tatsu-syo** [comment]
@iccfish In insider version this issue is possible.
See how it goes.

**Comment by iccfish** [comment]
Confirmed. Disable flight reconcile schedule task does fix this issue for me. Need more time to verify.
